builds/e2e/templates/e2e-run.yaml

```diff
@@ -38,9 +38,20 @@ steps:
     {
       $filter = 'Category=nestededge_isa95'
     }
+    elseif ($test_type -eq 'http_proxy')
```

Comment:
I think it's better to create a new group for tests to disable behind the proxy instead of disabling tests by name individually.

Reply:
Agreed on categorizing tests. I am still investigating some tests before adding test categories to filter them out.